homeassistant/components/sensor/simulated.py

```diff
@@ -84,11 +91,14 @@ def __init__(self, name, unit, amp, mean, period, phase, fwhm, seed):
         self._seed = seed
         self._random = Random(seed)  # A local seeded Random
         self._start_time = dt_util.utcnow()
+        self._absolute_time = absolute_time
         self._state = None
 
     def time_delta(self):
         """"Return the time delta."""
         dt0 = self._start_time
+        if self._absolute_time:
+            dt0 = datetime.datetime(1970, 1, 1, tzinfo=pytz.utc)
```

Comment:
Please use `homeassistant.util.dt.UTC` as tzinfo

Reply:
I've updated that now.